### DISCUSSION TITLE
'let' keyword

### DISCUSSION THREAD

**Discussion by elavy-harris** [nextgenhealthcare/connect#5167] [General]
Using 4.0.0, if I try to use the 'let' Javascript keyword, the editor underlines the following variable name as an error.  The validator doesn't complain, and it seems to execute without issues.

Is this a defect in the editor window itself?

**Comment by pacmano1** [comment]
It is. The mirth editor will highlight other not bad code also.

**Reply by elavy-harris** [reply]
Thank you.  Is there an open issue for this (and the other not bad code)?

**Reply by pacmano1** [reply]
You can search the issues posted just like anyone else can.   If you do find one "closed due to inactivity", perhaps comment on it / and or create a new one.
